doc/modules/cassandra/pages/architecture/storage-engine.adoc

```diff
@@ -18,19 +18,19 @@ The sequence of the steps in the write path:
 [[commit-log]]
 == Logging writes to commit logs
 
-When a write occurs, {cassandra} writes the data to a local append-only (https://cassandra.apache.org/_/glossary.html#commit-log)[commit log] on disk.
-This action provides xref:cassandra:managing/configuration/cass_yaml_file.adoc[configurable durability] by logging every write made to a {cassandra} node.
+When a write operation takes place, Cassandra records the data in a local append-only https://cassandra.apache.org/_/glossary.html#commit-log[commit log] on disk.
```

Comment:
@arrapraveen221 cant be this also referenced somehow differently as below line? This will not render correctly if it is previewed locally.

Reply:
yes glossary, everything is done when doing `ant artifacts`.